### ISSUE TITLE
Missing validation dataset

### ISSUE THREAD

**Issue by BelowZero000** [wlydlut/UHDDIP#1]
Hello, I recently come across this on Arxiv and I really want to try out the repos with the Image Deblurring but the UHD_Deblur dataset doesnt have the validation folder for training. Please add it and hope your paper gets added to CVF!

**Comment by wlydlut** [comment]
Thank you for recognizing our work.

We have used one test image as the validation set to reduce training time. So, you can put one image or all images in the test set in the validation path.
